### PR TITLE
Create a Compatibility Page

### PR DESCRIPTION
**Pull request by IsaacMarovitz**
- Adds a compatibility graph
    - Data is read from a pre-computed JSON on the compat repo (https://github.com/Ryujinx/Ryujinx-Games-List/pull/4682)
- Updates home page to query GitHub directly for star count and playable game count

https://github.com/Ryujinx/Ryujinx-Website/assets/42140194/802379df-d181-4358-8ba5-d799349a06e5

### CODE REVIEW

File: src/chartjs.d.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import type { TooltipPositionerFunction } from 'chart.js';
+
+declare module 'chart.js' {
+  interface TooltipPositionerMap {
+    cursor: TooltipPositionerFunction<ChartType>;
+  }
+}
```
Comment (TSRBerry) [inline src/chartjs.d.ts:7]: Why is that needed? Chart.js ships with typescript declarations.

File: src/views/CompatibilityPage.vue [hunk 2]
```diff
@@ -0,0 +1,213 @@
+<script lang="ts" setup>
+import axios from "axios";
+import { onMounted, ref } from "vue";
+import { useI18n } from "vue-i18n";
+import { Bar } from "vue-chartjs";
+import { Chart, CategoryScale, LinearScale, BarElement, Title, Legend, Tooltip, Colors, ChartData, ChartOptions } from 'chart.js'
```
Comment (AcK77) [inline src/views/CompatibilityPage.vue:6]:
```suggestion
import { Chart, CategoryScale, LinearScale, BarElement, Title, Legend, Tooltip, Colors, ChartData, ChartOptions } from "chart.js"
```

**Review by AcK77 — COMMENTED** [review]
Sometimes you use `'` and sometimes `"` is used, it could be nice to get consistency here.

**Comment by TSRBerry** [comment]
@IsaacMarovitz do you plan on addressing the comments here?

File: src/views/CompatibilityPage.vue [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+  color: string;
+  count: number;
+}
+
+function setData(this: any) {
+  chartData.value = ({
+    labels: ["label"],
+    datasets: tierData.value.map((tier: PlayableTier): ChartDataset<"bar", number[]> => ({
+      data: [tier.count],
+      label: t(`views.compatibility.${tier.localeKey}`),
```
Comment (AcK77) [inline src/views/CompatibilityPage.vue:60]:
```suggestion
      label: t("views.compatibility.${tier.localeKey}"),
```

**Review by AcK77 — APPROVED** [review]

File: src/views/CompatibilityPage.vue [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+  <div class="bg-white">
+    <div class="space-y-16 container xl:max-w-7xl mx-auto px-4 py-16 lg:px-8 lg:py-32">
+      <div class="text-center space-y-4">
+        <h2 class="text-3xl md:text-4xl font-extrabold mb-4">
+          {{ t("views.compatibility.title2") }}
+        </h2>
+        <h3 class="text-lg md:text-xl md:leading-relaxed font-medium text-gray-600 lg:w-2/3 mx-auto">
+          {{ t("views.compatibility.description2") }}
+        </h3>
+        <GButton href="https://github.com/Ryujinx/Ryujinx-Games-List/issues" rounded variant="sky" size="elg">
```
Comment (TSRBerry) [inline src/views/CompatibilityPage.vue:207]: Could the href here be made configurable?

File: src/views/HomePage.vue [hunk 5]
```diff
@@ -8,9 +8,60 @@ import {
...
+
+  try {
+    const result = await axios.get<Repo>(
+      import.meta.env.VITE_REPO_URL
+    );
+
+    repo.value = result.data;
+  } catch (err) {
+    console.error(err);
+  }
```
Comment (TSRBerry) [inline src/views/HomePage.vue:62]: These blocks all look very similar, could you write a helper function to simplify them?
